test/zzz-rsyslog.sh

```diff
@@ -136,12 +143,10 @@ deploy_using_operators() {
     sleep 10
     os::cmd::try_until_success "oc get cm rsyslog 2> /dev/null"
     # enable annotation_match
-    oc get cm rsyslog -o yaml | \
-      sed -e 's/action(type="mmkubernetes"/action(type="mmkubernetes" annotation_match=["."]/' | \
+    oc get cm rsyslog -o json | \
+      sed -e 's/action(type=\\"mmkubernetes\\"/action(type=\\"mmkubernetes\\" annotation_match=[\\".\\"]/' | \
```

Comment:
this meas the current code is not working?

Reply:
Yeah, it failed for me due to the invalid conversion from json to yaml...

Reply:
ah, I see - you changed `-o yaml` to `-o json`